Gemfile

```diff
@@ -6,12 +6,6 @@ ruby '3.1.3'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225281710-dd8784db-b92a-4ad6-a5de-2146b525c028.png)

**[Optional]**

- [ ] It would be nice if you make your Install, Usage, Run tests sub-sections look like below to make your readme file more descriptive :blush:

## Install
```
bundle install
rails db:create
rails db:migrate
```

## Usage
```
rails server or rails s
```

## Run tests
```
rspec spec
```